src/mlpack/tests/adaboost_test.cpp

```diff
@@ -319,13 +319,14 @@ BOOST_AUTO_TEST_CASE(HammingLossIris_DS)
   // Define your own weak learner, decision stumps in this case.
   const size_t numClasses = 3;
   const size_t inpBucketSize = 6;
-  DecisionStump<> ds(inputData, labels.row(0), numClasses, inpBucketSize);
+  arma::Row<size_t> labelsvec = labels.row(0);
```

Comment:
I see that you found a way to solve the issue, I still like to understand why this doesn't work in the first place, already reproduce the issue on my machine.

Reply:
My guess is that with `DecisionStump<>`, the labels were explicitly an `arma::Row<size_t>`, so the compiler automatically converted `labels.row(0)` (which actually has type `arma::subview_row<size_t>` or something like that).  But `ID3DecisionStump` allows any type for the labels.  So explicitly forming `arma::Row<size_t>` is now necessary, since the compiler would normally infer the wrong thing for `LabelsType` if we just passed `labels.row(0)`.